iothub/device/src/Common/Api/ClientApiVersionHelper.cs

```diff
@@ -6,7 +6,7 @@ namespace Microsoft.Azure.Devices.Client
     internal class ClientApiVersionHelper
     {
         internal const string ApiVersionQueryPrefix = "api-version=";
-        internal const string ApiVersionLatest = "2018-06-30";
+        internal const string ApiVersionLatest = "2019-10-01";
```

Comment:
Would this only effect device client? or is this expected to impact both device and service SDK's

Reply:
The service client can send desired property updates that contain string arrays now, too

Reply:
I added some tests for sending arrays in desired property updates, for reference